db/migrate/20221125083625_add_user_ref_to_posts.rb

```diff
@@ -0,0 +1,5 @@
+class AddUserRefToPosts < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :posts, :user, foreign_key: true
```

Comment:
- [ ]  according to project requirement, kindly reference user as author i.e `t.references :author, foreign_key: { to_table: 'users' } `